shared/editor/embeds/Grist.test.tsx

```diff
@@ -0,0 +1,40 @@
+import Grist from "./Grist";
+
+describe("Grist", () => {
+  const match = Grist.ENABLED[0];
+
+  test("to be enabled on share link", () => {
+    expect(
+      "https://templates.getgrist.com/doc/afterschool-program/p/2?embed=true".match(
+        match
+      )
+    ).toBeTruthy();
+    expect(
+      "https://docs.getgrist.com/sg5V93LuAije/Untitled-document/p/22?embed=true".match(
```

Comment:
Did you test this locally?

Why aren't the `ci` tests running against this PR? Any idea @tommoor ?

Reply:
Not sure, building of forks is enabled in CircleCI. Could be that @prio needs to follow the project there or something.

Reply:
> Did you test this locally?

Yes, tested it locally

Reply:
> Not sure, building of forks is enabled in CircleCI. Could be that @prio needs to follow the project there or something.

Ok, I am now following the project on circleci

Reply:
> Yes, tested it locally

I meant unit([`jest`](https://github.com/outline/outline/blob/0c51bfb8998a08e5565a9e9cd5c4f19a3fcd0df6/package.json#L27) ones), not manual testing. Anyways, good that CI started running tests.